index.js

```diff
@@ -55,10 +59,7 @@ module.exports = {
   postBuild(build) {
     this._super.included.apply(this, arguments);
 
-    const env = process.env.EMBER_ENV;
-    const ifaConfig = this.project.config(env).ifa;
```

Comment:
why did you remove the `ifaConfig` variable here? removing it means we run `this.project.config()` multiple times, which might be expensive in a few cases. maybe we should at least cache the result?